tests/unit/amo/api/test_users.js

```diff
@@ -166,4 +167,24 @@ describe(__filename, () => {
       mockApi.verify();
     });
   });
+
+  describe('deleteUserAccount', () => {
+    it('deletes a user profile', async () => {
+      const state = dispatchSignInActions().store.getState();
```

Comment:
Do we need to sign in for this test, or could you just create a `const` for `userId` and use that?

Reply:
we need the api state too.

Reply:
In other tests we use `dispatchClientMetadata().store.getState()` to get the state, which I suppose is six of one, half a dozen of the other, but if we don't explicitly need a signed in user, maybe we should use the former?

Reply:
I see. This action requires authentication so I'd keep a signed user because of that.